library/src/main/java/io/ona/kujaku/helpers/MapboxLocationComponentWrapper.java

```diff
@@ -0,0 +1,47 @@
+package io.ona.kujaku.helpers;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+
+import com.mapbox.android.core.location.LocationEngine;
+import com.mapbox.mapboxsdk.location.LocationComponent;
+import com.mapbox.mapboxsdk.location.modes.CameraMode;
+import com.mapbox.mapboxsdk.location.modes.RenderMode;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+
+import io.ona.kujaku.exceptions.LocationComponentInitializationException;
+
+/**
+ * @author Vincent Karuri
+ */
+public class MapboxLocationComponentWrapper {
```

Comment:
Could this class be moved to a function inside KukakuMapView as Its only initializing the location component.

Reply:
The idea behind the class is that if in future we need more control over the LocationComponent, this will be the place to enforce those policies.

KujakuMapView is already large and this avoids adding unnecessary bulk to it.